### ISSUE TITLE
Very nice preview! 

### ISSUE THREAD

**Issue by danielpetrica** [sondow77/Telegram-Theme-Creation#1]
Hi, I really love what you have done. 
Can I integrate it into my project?
My [Theme maker](https://github.com/danielpetrica/Telegram-Desktop-themes-maker) is missing such a nice preview.

**Comment by sondow77** [comment]
Yeah, you can integrate it, but I don't have done the previews, I can help you on my free time
